README.md

```diff
@@ -8,19 +8,42 @@ Simulators are needed to calculate OEE based on configuration profiles that can
 
 Prebuilt docker images can be downloaded from the [Releases](https://github.softwareag.com/IOTA/oee-simulators/releases) in this repository.
 
-# unit test 
+# Tests 
 Collection of test for the [Generic Simulators](simulators)
 
-In order to run all the test scripts:
+To run specific test script:
 ```
-python -m unittest discover -s test -p "*_test.py"
+python test/[script-name].py  [-h] [--tenant-id TENANT_ID] [--password PASSWORD] [--baseurl BASEURL] [--username USER]
 ```
-Run specific script:
+<pre>
+Cumulocity platform credentials setup
+
+optional arguments:<br>
+  long syntax                |    short syntax       |   Functions
+------------------------------------------------------------------------------------------
+  --help,                    |    -h                 |   show help message and exit
+  --tenant-id TENANT_ID,     |    -t TENANT_ID       |   Tenant ID
+  --password PASSWORD,       |    -p PASSWORD        |   C8Y Password
+  --baseurl BASEURL,         |    -b BASEURL         |   C8Y Baseurl
+  --username USER,           |    -u USER            |   C8Y Username
+
+It is important to note that <strong><ins>all four arguments: Tenant ID, C8Y Password, C8Y Baseurl, and C8Y Username must be filled</ins></strong>. 
+Failure to provide any of these fields may cause the script to malfunction or produce unexpected results.
+</pre>
+
+If you don't want to input arguments, you can set environment variables need to be set in [cumulocityAPI.py](simulators/main/cumulocityAPI.py) in order to run [simulators_test.py](test/simulators_test.py). <br>
+For example:
 ```
-cd test
-python -m unittest [test-script-name].py
+C8Y_BASEURL=https://perftest.2.performance.c8y.io 
+C8Y_TENANT=t3233
+C8Y_USER=viktor.tymoshenko@softwareag.com
```

Comment:
please avoid using a real username; change to `C8Y_USER=yourusername`

Reply:
I  just copied the default example in the README.md inside simulators directory. Let me change it